payments-ui-core/src/main/java/com/stripe/android/ui/core/PaymentsTheme.kt

```diff
@@ -43,6 +48,13 @@ object PaymentsThemeConfig {
         return if (isDark) colorsDark else colorsLight
     }
 
+    @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+    object Shapes {
```

Comment:
How is `Shapes` here compared to materials `Shapes`?

I am wondering if we should have our own `Shapes` val like we do [here](https://github.com/stripe/stripe-android/blob/master/link/src/main/java/com/stripe/android/link/theme/Shape.kt) and split out borderStrokeWidth into a `Dimens` object? I am fine with this approach though

Reply:
I think logically the radius and borders fall into the shapes bucket. This object is eventually going to be public and changeable so I want as little nesting as possible.